src/components/samples/SamplesListContent.js

```diff
@@ -47,27 +47,32 @@ const getTableColumns = (containersByID, individualsByID) => [
     },
     {
       title: "Individual",
-      dataIndex: "individual",
+      dataIndex: ["individual", "name"],
       sorter: true,
-      render: individual => (individual &&
+      render: (_, sample) => {
+        const individual = sample.individual
+        return (individual &&
           <Link to={`/individuals/${individual}`}>
             {withIndividual(individualsByID, individual, individual => individual.name, "loading...")}
-          </Link>),
+          </Link>)
+      }
     },
     {
       title: "Container Name",
-      dataIndex: "container_name",
+      dataIndex: ["container", "name"],
```

Comment:
Do you sometime need these as a list ? I read an earlier comment by Romain that pointed out that you could save yourself some hassles by story the info as string by using dataIndexToString at the source instead of each time it is used... I tend to agree with him that you would have a cleaner and safer code if you do not store this intermediate format.